#The fact, which was removed
string[] fact
#whether another fact exists in tuple space, which matches the requested fact
bool another_matching_fact_exists
